.generate.dart

```diff
@@ -0,0 +1,244 @@
+// **************************************************************************
+// Generator: Flutter boilerplate
+// Made by DNAMIC
+// **************************************************************************
+
+import 'dart:io';
+import 'package:yaml/yaml.dart';
+
+const source = 'lib/';
+const screen = 'screens/';
+
+Future<void> main(List<String> args) async {
```

Comment:
Very proud of this! 💙

Reply:
It's a team effort, I'm learning a lot working with you.